tests/cypress/plugins/imageGenerator/addPlugin.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+"use strict";
+
+Object.defineProperty(exports, "__esModule", {
+    value: true
+});
+exports.imageGenerator = imageGenerator;
+
+var jimp = require('jimp');
+var path = require('path');
+var fs = require('fs-extra');
+
+function imageGenerator(args) {
+    var directory = args.directory;
+    var fileName = args.fileName;
+    var width = args.width;
+    var height = args.height;
+    var color = args.color;
+    var message = args.message;
+    var image = new jimp(width, height, color, function (err, image) {
+        if (err) throw err;
+    });
```

Comment:
I suspect this function to be async. It is why it has callback as the latest argument. 
From this point of view it would be more reliable to do all actions (printing text, and saving it) with the image inside the callback.